packages/atlas/src/components/Avatar/Avatar.styles.ts

```diff
@@ -180,11 +179,15 @@ export const Container = styled('div', { shouldForwardProp: isPropValid })<Conta
   ${getBorderStyles};
 
   border-radius: 100%;
+  padding: 0;
+  border: 0;
+  background: none;
+
   display: flex;
   justify-content: center;
   align-items: center;
   position: relative;
-  cursor: ${({ onClick }) => (onClick ? 'pointer' : 'unset')};
+  cursor: ${({ isClickable }) => (isClickable ? 'pointer' : 'inherit')};
```

Comment:
WDYT?
```suggestion
  cursor: ${({ isClickable }) => (isClickable ? 'pointer' : 'default')};
```

Reply:
Would this help with some issue? If we use `default`, it may override inherited styled. For example, if you nest avatar inside a link, with `default` it wouldn't inherit the pointer